src/util/router/pages.tsx

```diff
@@ -216,6 +216,7 @@ export const pages: PageType = {
     },
     route: {
       path: "explorer/",
+      handle: { explorer: true },
```

Comment:
What do these properties in "handle" represent exactly? I wonder if the name (`explorer` in this case) could be changed to something containing more semantic information.

Reply:
The name explorer in this case must exactly match the key of the page, so that the Navigation component has one programmatic way to detect the active (root) menu

`const isActive = checkHandlerInMatcher(secondPathLevel, key);`